catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/table/thead.less

```diff
@@ -48,6 +48,20 @@
       }
     }
 
+    .ui-resizable {
+       text-align: right;
```

Comment:
❓ Why align right?

Reply:
I don't think this is necessary anymore, but aligning it to the right made it easier to grab the handle for resize because it would be aligned to the right (just in case it got weird when resizing).